scripts/macos-gen-dmg.py

```diff
@@ -60,4 +70,9 @@ def main(args):
 
 
 if __name__ == '__main__':
-    run_main(main)
+    try:
+        run_main(main)
+    except subprocess.CalledProcessError as e:
+        print("main dmg creation method failed, trying backup")
+        import sys
+        main(sys.argv)
```

Comment:
I don't understand the purpose of this. What's the "backup" and how is this supposed to work?